bin/yaml/asmx.yaml

```diff
@@ -0,0 +1,9 @@
+compilers:
+  asmx:
+    arm:
```

Comment:
No, I looked at other YAML examples. I have indicated that the user can compile Asm programs in ARM.

Reply:
I forgot, this is the interpretation version so far, without the release of a new file. Later I will make version 3 where it will already be possible to Compile under ARM. Right now I don't have an archive for that yet.

Reply:
In general, I will wait until NodeJS and npm support is available.

Reply:
got it: if the compiler isn't specifically for arm, then probably drop this. Appreciate that you checked others, the "path" (compilers/asmx/arm) is the 'name" of the compiler for the purposes of installing it, but not otherwise user visible.

Reply:
You previously said that you support interpretive programming languages like Python.

Reply:
We do: what does that have to do with the request to remove `arm` from this?

Reply:
should I replace `arm` with `asmx`?

Reply:
Unless this is specifically an `arm` one, then it doesn't need a specific entry here at all. The keys are inherited by nesting...so I think this whole file is just:
```yaml
compilers:
  asmx:
    type: tarballs
    compression: gz
    url: https://github.com/langprogramming-AsmX/AsmX/releases/download/v{name}AsmX-v{name}.tar.gz
    check_exe: asmx-cli -v
    targets:
    - '2.0'
    - '3.0.0'
 ```

The idea is the yaml file reduces redundancy. If you _needed_ a whole separate installation configuration for "arm", you could nest an `arm:` thing somewhere, but I think this is all you need :)